app/serializers/api/v1/offer_summary_serializer.rb

```diff
@@ -7,7 +7,7 @@ class OfferSummarySerializer < ApplicationSerializer
       :received_at, :cancelled_at, :start_receiving_at,
       :submitted_items_count, :accepted_items_count, :rejected_items_count,
       :expecting_packages_count, :missing_packages_count, :received_packages_count,
-      :display_image_cloudinary_id, :notes
+      :display_image_cloudinary_id, :notes, :inventoried_package_count, :unrecorded_package_count
```

Comment:
How are those fields used ? 🤔

Reply:
On Offer search we have to show these count beside offer name.

Reply:
As discussed in slack, abul will remove this as it was an unexpected artifact from the mockup